documentation/api/query/v4/reports.markdown

```diff
@@ -122,42 +153,152 @@ Query for all reports:
+        "message" : "defined 'message' as 'hi world'"
       }, {
-        "containment_path" : [ "Stage[main]", "Main", "File[/home/wyatt/Desktop/foo]" ],
-        "new_value" : "\"file\"",
-        "resource_title" : "/home/wyatt/Desktop/foo",
-        "property" : "ensure",
+        "new_value" : "hi world",
+        "property" : "message",
         "file" : "/home/wyatt/.puppet/manifests/site.pp",
-        "old_value" : "\"absent\"",
-        "line" : 7,
-        "status" : "changed",
-        "resource_type" : "File",
-        "timestamp" : "2014-12-24T00:00:50.514Z",
-        "message" : "defined content as '{md5}207995b58ba1956b97028ebb2f8caeba'"
+        "old_value" : "absent",
+        "line" : 3,
+        "resource_type" : "Notify",
+        "status" : "success",
+        "resource_title" : "hi",
+        "timestamp" : "2015-02-19T16:23:10.767Z",
+        "containment_path" : [ "Stage[main]", "Main", "Notify[hi]" ],
+        "message" : "defined 'message' as 'hi world'"
+      } ],
+      "status" : "changed",
+      "configuration_version" : "1424362990",
+      "environment" : "production",
+      "certname" : "desktop.localdomain",
+      "metrics" : [ {
+        "category" : "resources",
+        "name" : "changed",
+        "value" : 2
+      }, {
+        "category" : "resources",
+        "name" : "failed",
+        "value" : 0
+      }, {
+        "category" : "resources",
+        "name" : "failed_to_restart",
+        "value" : 0
+      }, {
+        "category" : "resources",
+        "name" : "out_of_sync",
+        "value" : 2
+      }, {
+        "category" : "resources",
+        "name" : "restarted",
+        "value" : 0
+      }, {
+        "category" : "resources",
+        "name" : "scheduled",
+        "value" : 0
+      }, {
+        "category" : "resources",
+        "name" : "skipped",
+        "value" : 0
+      }, {
+        "category" : "resources",
+        "name" : "total",
+        "value" : 9
+      }, {
+        "category" : "time",
+        "name" : "config_retrieval",
+        "value" : 0.476064209
+      }, {
+        "category" : "time",
+        "name" : "filebucket",
+        "value" : 3.8841E-5
+      }, {
+        "category" : "time",
+        "name" : "notify",
+        "value" : 7.54224E-4
+      }, {
+        "category" : "time",
+        "name" : "schedule",
+        "value" : 2.0780000000000004E-4
+      }, {
+        "category" : "time",
+        "name" : "total",
+        "value" : 0.47706507400000003
+      }, {
+        "category" : "changes",
+        "name" : "total",
+        "value" : 2
+      }, {
+        "category" : "events",
+        "name" : "failure",
+        "value" : 0
+      }, {
+        "category" : "events",
+        "name" : "success",
+        "value" : 2
+      }, {
+        "category" : "events",
+        "name" : "total",
+        "value" : 2
+      } ],
+      "logs" : [ {
+          "file" : null,
```

Comment:
while I'm being picky about whitespace in the examples, these properties (and the rest in this array) should be indented left 2 spaces to be consistent :-).